Cargo.toml

```diff
@@ -8,11 +8,12 @@ name = "rusl"
 crate-type = ["staticlib", "dylib"]
 
 [dependencies]
-lazy_static = { version = "0.2.1", features = ["spin_no_std"] }
+lazy_static = { version = "0.2.2", features = ["spin_no_std"] }
 rlibc = "1.0.0"
-spin = "0.4.0"
-syscall = "0.2.1"
-va_list = "0.0.3"
+spin = "0.4.5"
+sc = "0.1.5"
```

Comment:
kmcallister's project seems stagnant. `steed` in fact uses `sc` (same person behind both), which is why I thought it'd be ok to move to `sc` from `syscall`.